source/hardware/controllers/akai_apc_mini.rst

```diff
@@ -0,0 +1,20 @@
+AKAI APC Mini
```

Comment:
I think the manufacturer now uses "Akai" instead of all-uppercase letters in written text.

```suggestion
Akai APC Mini
```

Reply:
On the website I seen AKAI always written uppercase, it' printed uppercase on the product themselves, but in the FAQ they write Akai... they need to make peace with their own brains.
Anyway, let's write it lowercase :)